store/mgostore/store.go

```diff
@@ -53,6 +53,13 @@ func (s *identityStore) Identity(ctx context.Context, identity *store.Identity)
 	identity.ProviderInfo = doc.ProviderInfo
 	identity.ExtraInfo = doc.ExtraInfo
 	identity.Owner = store.ProviderIdentity(doc.Owner)
+
+	var err error
+	identity.Credentials, err = s.UserMFACredentials(ctx, doc.ProviderID)
```

Comment:
This seems to be a very strange special case. Why not make these credentials part of the identity document? then it would not need a second query to get them. Also there wouldn't be a difference between the returned values for `Identity` and `FindIdentities`

Reply:
removed